File: Extras/CodeChef/CompetitionPractice/CodeChefStarter21/CovidAndTheaterTickets/Codechef.java
#include <iostream>
#include <cmath>
using namespace std;
//Problem Code : COVID_19
int main() {
	int t;//no of testcases 
	cin>>t;
	while(t--){
	    double m,n;
	    cin>>m>>n;
	    int res = ceil(m/2) * ceil(n/2);
	    cout<<res<<endl;
	}
	return 0;
}

File: Extras/CodeChef/CompetitionPractice/Starters34_13Apr22/TRUESCORE/Codechef.cpp
#include <iostream>
using namespace std;

int main() {
	int t=0;
	cin>>t;
	while(t--){
	    int a[2];
	    cin>>a[0]>>a[1];
	    int b[2];
	    cin>>b[0]>>b[1];
	    if(b[0]<a[0] || b[1]<a[1])cout<<"IMPOSSIBLE"<<endl;
	    else cout<<"POSSIBLE"<<endl;
	}
	
	return 0;
}

